packages/web/src/pages/upload-page/forms/EditTrackForm.module.css

```diff
@@ -1,6 +1,7 @@
 .formContainer {
   width: 100%;
   background-color: var(--white);
+  min-width: 640px;
```

Comment:
wondering if we want different min-widths for edit track and multi-track (considering the sidebar)? maybe check with Julian